src/app/app.component.scss

```diff
@@ -55,7 +55,7 @@ body {
     ion-content {
         --padding-start: 2rem;
         --padding-end: 2rem;
-        width: calc(100% + 15px);
+        // width: calc(100% + 15px);
```

Comment:
remove unused code

Reply:
add TODO: to check scroll if wrong in future